src/pages/invoices/create/Create.tsx

```diff
@@ -95,7 +95,10 @@ export function Create() {
     setInvoice((current) => {
       let value = current;
 
-      if (searchParams.get('action') !== 'clone') {
+      if (
+        searchParams.get('action') !== 'clone' &&
+        searchParams.get('action') !== 'invoice'
+      ) {
```

Comment:
Naming this is wrong. The invoice page should never have the action `invoice` because that's what the default way of doing something is. 

Can you elaborate on this, please?

Reply:
@beganovich So the point is the same as for the `clone` action. Through `project invoicing`, we set the invoice atom to some value calculated in the `useInvoiceProject` hook. If we do not add this part of the condition we will always have an `undefined` invoice value. But I absolutely agree with the naming, I didn't get it, I renamed the action to `project_invoicing`, I hope you agree with this naming too. Let me know your thoughts.